proto/settings.proto

```diff
@@ -9,4 +9,6 @@ message SettingsMsg {
   string ap_self_password = 6;
   string ap_self_name = 7;
   int32 wifi_power = 8;
+  bool board_locked = 9;
```

Comment:
The names of these variables are a little ambiguous. s/board_locked/board_locking_enabled/ ?

Maybe add comments to these fields to explain what exactly they stand for?